Dockerfile

```diff
@@ -28,7 +28,7 @@ RUN git clone https://github.com/mendersoftware/mender.git . && \
     git checkout $MENDER_CLIENT_VERSION && \
     DESTDIR=/install-modules-gen make install-modules-gen
 
-FROM docker:25-dind
+FROM docker:26-cli
```

Comment:
I have absolutely no idea will it work? I mean: how many things will it break? cli and dind are vastly different.

Reply:
Exactly! It broke because we were using the docker-in-docker privileged image which has a scripted entrypoint. I'm merely changing the image back to alpine with docker installed.
Also, please see the ticket for a longer explanation for this change and the caveats.